src/ledger-keyring.ts

```diff
@@ -212,7 +212,9 @@ export class LedgerKeyring extends EventEmitter {
         hdPath: path,
       });
     } catch (error) {
-      throw error instanceof Error ? error : new Error('Unknown error');
+      throw error instanceof Error
+        ? error
+        : new Error('Unlock your Ledger device and open the ETH app');
```

Comment:
How about this message?

```suggestion
        : new Error('Ledger ETH app is closed. Open it to unlock.');
```

Reply:
The current error message was defined by Vivek, and for some reason, Extension will throw `Unknown Error` even ledger is locked. that is why we have Unlock your ledger first in error message, and then ask user to open ETH app.

Reply:
Is this error being used in the UI?

Reply:
yes, this error will display in `select hardware screen` in metamask extension. i dont know why we didn't do the localisation translation for those error messsage in frontend.

Reply:
@AlexJupiter any preference which we shall use?

Reply:
> I don't know why we didn't do the localisation translation for those error messsage in frontend.

This was my point, user facing errors should be added to the files for translation. This error is just for developers I recommend to use `Ledger Ethereum app is closed. Open it to unlock.` for development error purposes and catch this error in the UI level to show a user friendly message.

Overall I agree with @dawnseeker8 approach regarding this

Reply:
i have changed the message to `Ledger Ethereum app closed. Open it to unlock.`